client/app/components/group/groupDetail/_groupOptions/groupOptions.html

```diff
@@ -54,6 +54,12 @@
     </md-button>
     <md-menu-content>
       <md-menu-item>
+          <md-button ui-sref="groupInvites" aria-label="{{ 'GROUP.INVITE_TITLE' | translate }}">
```

Comment:
This has 4 spaces indentation, while we usually have 2 spaces. Can you check your editor setting?